packages/lab/src/components/Flow/Node/Node.tsx

```diff
@@ -59,63 +64,58 @@ const isInputConnected = (id, type, idx, edges) => {
       (e) => e.source === id && e.sourceHandle === idx.toString()
     );
   }
+
+  return false;
 };
 
+const defaultActions: HvFlowNodeAction[] = [
```

Comment:
Since these are now the default actions for the node, we can remove `defaultActions` from our stories.